code/iaas/lb-instance/src/main/java/io/cattle/platform/lb/instance/process/LoadBalancerAddHostPostAction.java

```diff
@@ -0,0 +1,47 @@
+package io.cattle.platform.lb.instance.process;
+
+import io.cattle.platform.core.constants.LoadBalancerConstants;
+import io.cattle.platform.core.model.LoadBalancer;
+import io.cattle.platform.core.model.LoadBalancerHostMap;
+import io.cattle.platform.engine.handler.HandlerResult;
+import io.cattle.platform.engine.handler.ProcessPostListener;
+import io.cattle.platform.engine.process.ProcessInstance;
+import io.cattle.platform.engine.process.ProcessState;
+import io.cattle.platform.lb.instance.service.LoadBalancerInstanceManager;
+import io.cattle.platform.process.common.handler.AbstractObjectProcessLogic;
+import io.cattle.platform.util.type.Priority;
+
+import javax.inject.Inject;
+import javax.inject.Named;
+
+@Named
+public class LoadBalancerAddHostPostAction extends AbstractObjectProcessLogic implements ProcessPostListener, Priority {
+
+    LoadBalancerInstanceManager lbInstanceManager;
+
+    @Override
+    public String[] getProcessNames() {
+        return new String[] { LoadBalancerConstants.PROCESS_LB_HOST_MAP_CREATE };
+    }
+
+    @Override
+    public HandlerResult handle(ProcessState state, ProcessInstance process) {
+        LoadBalancerHostMap map = (LoadBalancerHostMap) state.getResource();
+        LoadBalancer lb = loadResource(LoadBalancer.class, map.getLoadBalancerId());
+        long hostId = map.getHostId();
+
+        lbInstanceManager.getLoadBalancerInstances(lb, hostId);
```

Comment:
Makes a call to lbInstanceManager to create LB instance (if it doesn't exist already). Method name is misleading I guess, I will rename it to "createLoadBalancerInstance"